lib/gstreamer/encoderbase.py

```diff
@@ -226,3 +228,15 @@ def md5_demuxed(self):
       get_media()._purge_test_artifact(demuxed)
       return result
     return md5(self.encoder.encoded)
+
+  def check_max_frame_size(self):
+    if vars(self).get("maxFrameSize", None) is None:
+      return
+
+    output = call(
+      f"{exe2os('ffprobe')}"
+      " -i {encoder.encoded} -show_frames | grep pkt_size".format(**vars(self)))
```

Comment:
We can't assume ffprobe is available in gstreamer tests.  Our CI does not install ffmpeg when it runs the gstreamer tests.